internal/graphql/resolver/operator.go

```diff
@@ -25,78 +28,85 @@ const (
 
 func (r *Resolver) changeOperatorStatus(ctx context.Context, status *model.Status) (*model.Status, error) {
 	delete := true
+	st := model.StatusProcessing
 	if *status == model.StatusEnabled {
 		delete = false
 	}
 
-	go func(del bool) {
-		kubeClient, err := r.GetKubeClient()
-		if err != nil {
-			r.operatorChannel <- &model.OperatorStatus{
-				Status: status,
-				Error: &model.Error{
-					Code:        errCode,
-					Description: "failed to create meshkube client",
-				},
-			}
-			return
-		}
+	if r.KubeClient.KubeClient == nil {
+		r.Log.Error(ErrNilClient)
+		return nil, ErrNilClient
+	}
 
-		er := initialize(kubeClient, del)
-		if er != nil {
+	go func(del bool, kubeclient *mesherykube.Client) {
+		err := initialize(kubeclient, del)
+		if err != nil {
+			r.Log.Error(err)
 			r.operatorChannel <- &model.OperatorStatus{
 				Status: status,
 				Error: &model.Error{
-					Code:        errCode,
-					Description: er.Error(),
+					Code:        "",
+					Description: err.Error(),
 				},
 			}
 			return
 		}
+		r.Log.Info("Initialized")
 
 		r.meshsyncChannel = make(chan *broker.Message)
-		err = subscribeToBroker(kubeClient, r.meshsyncChannel)
+
+		endpoint, err := subscribeToBroker(kubeclient, r.meshsyncChannel)
 		if err != nil {
+			r.Log.Error(err)
+			r.Log.Info(endpoint)
 			r.operatorChannel <- &model.OperatorStatus{
 				Status: status,
 				Error: &model.Error{
-					Code:        errCode,
+					Code:        "",
 					Description: err.Error(),
 				},
 			}
 			return
 		}
+		r.Log.Info("Connected to broker at:", endpoint)
 
 		// installMeshsync
-		err = runMeshSync(kubeClient, del)
+		err = runMeshSync(kubeclient, del)
 		if err != nil {
+			r.Log.Error(err)
 			r.operatorChannel <- &model.OperatorStatus{
 				Status: status,
 				Error: &model.Error{
-					Code:        errCode,
+					Code:        "",
 					Description: err.Error(),
```

Comment:
"MeshSync failed to start"?

Reply:
Oh actually the same case as above, before starting with messaging system implementation, its better to have it blank. To avoid confusion. wt say? @leecalcote